### DISCUSSION TITLE
API sample data Create Domain

### DISCUSSION THREAD

**Discussion by imbraintl** [modoboa/modoboa#2748] [Help]
Good day,
I am trying to create a domain via API, but am not able ase it says:
{
  "transport": {
    "service": [
      "This field may not be blank."
    ]
  }
}

May someone help me with sample data as i seem not to get it

{
  "name": "tested.com",
  "quota": "5000",
  "default_mailbox_quota": "0",
  "enabled": true,
  "type": "Simple User",
  "enable_dkim": false,
  "dkim_key_selector": "",
  "dkim_key_length": 1024,
  "dkim_private_key_path": "",
  "message_limit": 0,
  "domain_admin": {
    "username": "",
    "password": "",
    "with_mailbox": false,
    "with_aliases": false
  },
  "transport": {
    "service": "",
    "settings": {
      "additionalProp1": "",
      "additionalProp2": "",
      "additionalProp3": ""
    }
  }
}

**Comment by tonioo** [comment]
The value you put in the type field is wrong. You should use either "domain" or "relay".

**Reply by imbraintl** [reply]
Hey, that is not the issue, i may have mistyped it hey.
The problem is:
{
"transport": {
"service": [
"This field may not be blank."
]
}
}
What should i put in the "transport": {
"service": "",
"settings": {
"additionalProp1": "",
"additionalProp2": "",
"additionalProp3": ""
}

**Reply by tonioo** [reply]
You don't need the transport part is you set type to 'domain'
